examples/nifpga/read_write_example.py

```diff
@@ -0,0 +1,187 @@
+r""" Read and Write to indicators, controls, Arrays and FIFO.
+
+The gRPC API is built from the C API. NI-FPGA documentation is installed with the driver at:
+    C:\Program Files (x86)\National Instruments\FPGA Interface C API\capi.chm
+
+Getting Started:
+
+To run this example, you need to provide the correct file path and bitfile signature.
+Additionally, you should have a VI that includes controls, arrays, and FIFO,
+and update the references for them below. Ensure that the R-Series drivers are
+installed on which the server would run. You can install FPGA C Interface API as well.
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC
+Client" wiki page:
+  https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Run from command line as we need to pass Resource Name:
```

Comment:
When creating a new FPGA target, the resource name is initially empty (""). If you attempt to compile any VI with this empty resource name, the system will prompt you to select a resource.
Typically, the resource name for an FPGA in your system follows the format PXI1SlotX, where X is the slot number (e.g., PXI1Slot13).
To make sure it's proper, we need to specify a valid target/resource name. Users must update this name to match the actual FPGA resource in their system; otherwise, the compilation process will not work or we'll get an error when creating a session itself.

Reply:
Removed the comment.